packages/shared/src/proj/projection.ts

```diff
@@ -73,38 +73,47 @@ export class Projection {
     projectMultipolygon(multipoly: Position[][][], targetProjection: Projection): Position[][][] {
         if (targetProjection.epsg.code === this.epsg.code) return multipoly;
 
-        const { toWsg84 } = this;
-        const { fromWsg84 } = targetProjection;
+        const { toWgs84 } = this;
+        const { fromWgs84 } = targetProjection;
 
-        return multipoly.map((poly) => poly.map((ring) => ring.map((p) => fromWsg84(toWsg84(p)))));
+        return multipoly.map((poly) => poly.map((ring) => ring.map((p) => fromWgs84(toWgs84(p)))));
     }
 
     /**
      * Convert source `[x, y]` coordinates to `[lon, lat]`
      */
-    get toWsg84(): (coordinates: Position) => Position {
+    get toWgs84(): (coordinates: Position) => Position {
         return this.projection.forward;
     }
 
     /**
      * Convert `[lon, lat]` coordinates to source `[x, y]`
      */
-    get fromWsg84(): (coordinates: Position) => Position {
+    get fromWgs84(): (coordinates: Position) => Position {
         return this.projection.inverse;
     }
 
+    boundsToWgs84(source: Bounds): Bounds {
+        const sw = this.toWgs84([source.x, source.y]);
+        const ne = this.toWgs84([source.right, source.bottom]);
+
+        const width = ne[0] - sw[0];
+
+        return new Bounds(sw[0], sw[1], width < 0 ? width + 360 : width, ne[1] - sw[1]);
```

Comment:
why normalize only one side?

should also be done inside of a while loop to properly normalize them

Reply:
What else is there to normalize?

Projections will never span more than the entire glob so no need to be in a loop.

Reply:
One think to check with QGIS and ArcGIS is if bounds over the Anti-merdian are acceptable. e.g

<ows:WGS84BoundingBox>
<ows:LowerCorner>165.0 -50.0</ows:LowerCorner>
<ows:UpperCorner>185.0 -25.0</ows:UpperCorner>
</ows:WGS84BoundingBox>

I know in WFS this works ok. But likely the WMTS ArcGIS codebase is completely different.

Reply:
ATM it will revert to globe extent if bounds crosses the anti-meridian

Reply:
> Projections will never span more than the entire glob so no need to be in a loop.

Is this one of those you will never need more than 64k of ram sort of comments :P